src/Sylius/Bundle/ApiBundle/Resources/config/api_resources/OrderItem.xml

```diff
@@ -18,7 +18,7 @@
     <resource class="%sylius.model.order_item.class%" shortName="OrderItem">
         <attribute name="normalization_context">
             <attribute name="groups">
-                <attribute>order_item:read</attribute>
+                <attribute>admin:order_item:read</attribute>
```

Comment:
Instead of leaving the generic group here, I would prefer to do it explicitly on each action
```suggestion
```